### PR TITLE
[DOCS] Editing and Markup Adjustments

### PR DESCRIPTION
**Pull request by RobinJoe**
**Summary:** This change fixes a broken procedure markup, and adds some missing words.

**Details, and why the changes are needed:**
- The procedure "Create an Image Store" was not displaying the step numbers correctly
- Missing words: “an” L571
- Adjusting the markup of file pathways to match the other file pathways presented in the document.
- Change to active voice in an *important* admonition
- Removing a typo: “of” in a Troubleshooting section
- Changed “investigation” to “investigating” for a stronger verb, and changed “usual checks” to
  “these are checks you can try” since not all administrators may  know what is and is not usual
  for checking image build logs.

**For Reviewers:** There are additional words and some active voice adjustments to make, but this is a start. Please review and let me know about further changes.

### CODE REVIEW

**Comment by RobinJoe** [comment]
Hi @Loquacity , @jcayouette , and @keichwa , I have some editing in the Uyuni Administrator guide in this commit.

File: modules/administration/pages/image-management.adoc [hunk 1]
```diff
@@ -470,13 +470,18 @@ However if the certificate path was changed or unavailable, you will need to cal
 /usr/sbin/mgr-package-rpm-certificate-osimage --ca-cert-full-path /root/ssl-build/RHN-ORG-TRUSTED-SSL-CERT
 ----
 
-The RPM package with the certificate is stored in a salt-accessible directory such as `/usr/share/susemanager/salt/images/rhn-org-trusted-ssl-cert-osimage-1.0-1.noarch.rpm`.
-
-The RPM package with the certificate is provided in the local build host repository `/var/lib/Kiwi/repo`.
+The RPM package with the certificate is stored in a salt-accessible directory such as:
+----
+/usr/share/susemanager/salt/images/rhn-org-trusted-ssl-cert-osimage-1.0-1.noarch.rpm.
```
Comment (keichwa) [inline modules/administration/pages/image-management.adoc:475]:
```suggestion
/usr/share/susemanager/salt/images/rhn-org-trusted-ssl-cert-osimage-1.0-1.noarch.rpm
```

File: modules/administration/pages/image-management.adoc [hunk 2]
```diff
@@ -470,13 +470,18 @@ However if the certificate path was changed or unavailable, you will need to cal
...
+----
 
+The RPM package with the certificate is provided in the local build host repository:
+----
+/var/lib/Kiwi/repo.
```
Comment (keichwa) [inline modules/administration/pages/image-management.adoc:480]:
```suggestion
/var/lib/Kiwi/repo
```

File: modules/administration/pages/image-management.adoc [hunk 3]
```diff
@@ -470,13 +470,18 @@ However if the certificate path was changed or unavailable, you will need to cal
...
+----
 
 [IMPORTANT]
-.The RPM Package with the {productname} Certificate Must Be Specified in the Build Source
+.Specify the RPM Package with the {productname} Certificate in the Build Source
```
Comment (Loquacity) [inline modules/administration/pages/image-management.adoc:484]: We don't put a title on admonitions, because we don't have a proper style implemented for them (yet!). I would move this information in to be the first line in the admon, rather than a title.

File: modules/administration/pages/image-management.adoc [hunk 4]
```diff
@@ -563,7 +568,7 @@ Activation keys ensure that images using a profile are assigned to the correct c
 [NOTE]
 .Relationship Between Activation Keys and Image Profiles
```
Comment (Loquacity) [inline modules/administration/pages/image-management.adoc:569]: Drive-by: Remove this title too.

**Comment by RobinJoe** [comment]
Copying the commit message here for easier reading:

**Summary:** This change removes and rewrites admonition titles and typos

**Details, and why the changes are needed:**

- Removing the titles of admonitions, and including the information inside the
  admonition as suggested by reviewers.
- Changing a typo: comand to command
- changing filesystem to file system following the wug.

**For Reviewers:** Changes completed, further nitpicks are welcome, please let me
know of any further changes.

File: modules/administration/pages/image-management.adoc [hunk 5]
```diff
@@ -17,7 +17,7 @@ The workflow looks like this:
 {productname} supports two distinct build types: dockerfile, and the Kiwi image system.
 
 The Kiwi build type is used to build system, virtual, and other images.
-The image store for the Kiwi build type is pre-defined as a filesystem directory at [path]``/srv/www/os-images`` on the server.
+The image store for the Kiwi build type is pre-defined as a file system directory at [path]``/srv/www/os-images`` on the server.
```
Comment (Loquacity) [inline modules/administration/pages/image-management.adoc:20]: this probably needs to be done suite-wide :woman_facepalming:

File: modules/administration/pages/image-management.adoc [hunk 6]
```diff
@@ -404,9 +401,8 @@ Kiwi image configuration files and configuration scripts must be accessible in o
 For an example of a complete Kiwi repository served by git, see https://github.com/SUSE/manager-build-profiles/tree/master/OSImage
 
 [NOTE]
-.Hardware Requirements for Hosts Running OS Images
 ====
-Hosts running OS Images built with Kiwi need at least 1 GB of RAM.
+You will need at least 1GB of RAM available for Hosts running OS Images built with Kiwi.
```
Comment (Loquacity) [inline modules/administration/pages/image-management.adoc:405]:
```suggestion
You will need at least 1{nbsp}GB of RAM available for Hosts running OS Images built with Kiwi.
```

Another inconsistency across the whole suite, I think!

File: modules/administration/pages/image-management.adoc [hunk 7]
```diff
@@ -470,15 +466,20 @@ However if the certificate path was changed or unavailable, you will need to cal
...
+----
+/var/lib/Kiwi/repo
+----
 
 [IMPORTANT]
-.The RPM Package with the {productname} Certificate Must Be Specified in the Build Source
 ====
-Make sure your build source Kiwi configuration contains `rhn-org-trusted-ssl-cert-osimage` as a required package in the `bootstrap` section.
+Specify the RPM Package with the {productname} Certificate in the Build source,
+and make sure your build source Kiwi configuration contains `rhn-org-trusted-ssl-cert-osimage` as a required package in the `bootstrap` section.
```
Comment (Loquacity) [inline modules/administration/pages/image-management.adoc:482]:
```suggestion
Specify the RPM package with the {productname} SSL certificate in the build source, and make sure your Kiwi configuration contains ``rhn-org-trusted-ssl-cert-osimage`` as a required package in the ``bootstrap`` section.
```

File: modules/administration/pages/image-management.adoc [hunk 8]
```diff
@@ -561,9 +561,8 @@ Activation keys ensure that images using a profile are assigned to the correct c
 +
 
 [NOTE]
-.Relationship Between Activation Keys and Image Profiles
 ====
-When you associate activation key with an image profile you are ensuring any image using the profile will use the correct software channel and any packages in the channel.
+Associate an activation key with an image profile to ensure the image profile uses the correct software channel, and any packages in the software channel.
```
Comment (Loquacity) [inline modules/administration/pages/image-management.adoc:565]:
```suggestion
Associate an activation key with an image profile to ensure the image profile uses the correct software channel, and any packages.
```

File: modules/administration/pages/image-management.adoc (hunk 5 above)
Comment (RobinJoe) [reply modules/administration/pages/image-management.adoc:20]: I can use sed and grep to find the instances of filesystem, and then a script to find and replace them in each file.  Seems like something I should check out an issue with a plan of how I can write something to find and replace the use in text.

Comment (Loquacity) [reply modules/administration/pages/image-management.adoc:20]: Sure, go right ahead.

**Comment by Loquacity** [comment]
Happy for you to merge at will! :smile:

**Comment by Loquacity** [comment]
> Happy for you to squash and merge at will!

@RobinJoe doesn't have the perms to do this, so I'm going to go ahead.
